src/Http/Part.php

```diff
@@ -45,6 +45,13 @@ class Part extends TreeRouteStack implements RouteInterface
      */
     protected $childRoutes;
 
+    /**
+     * Priority.
+     *
+     * @var int|null
+     */
+    public $priority;
```

Comment:
Is it supposed to be public? 🤔

Reply:
it seems yes

https://github.com/laminas/laminas-router/blob/ef9f328862aac9492b19448f38b3ad866fb62932/src/Http/TreeRouteStack.php#L228

Reply:
Annoying but okay 🤔

I think we should then roll with this, by perhaps marking the property as `@internal`, and explaining why it was added.

Alternatively, a better fix would be to remove the property assignment overall, but unsure if that is a BC issue

Reply:
Afaik, all undefined properties which are being set are automatically public: https://3v4l.org/k2KdH

Reply:
@matbech the CS fixer is complaining a bit here: could you run `vendor/bin/phpcbf` to see if it can fix the annotation group for you?